lib/galaxy/config/__init__.py

```diff
@@ -732,6 +716,14 @@ def _process_config(self, kwargs):
                 'filters': ['stack']
             }
 
+    def _load_list_from_file(filepath):
+        try:
+            with open(filepath) as f:
+                return [line.strip() for line in f]
+        except IOError:
+            log.error("CONFIGURATION ERROR: Can't open supplied file: %s", filepath)
+            raise
```

Comment:
I would simplify to:
```suggestion
        with open(filepath) as f:
            return [line.strip() for line in f]
```